lute/tasks/sfx_index.py

```diff
@@ -0,0 +1,49 @@
+"""
+Classes for indexing tasks in SFX.
+
+Classes:
+    MergeStreamFIles: task that merges multiple stream files into a single file.
+"""
+
+__all__ = ["MergeStreamFiles"]
+__author__ = "Valerio Mariani"
+
+import shutil
+import sys
+from pathlib import Path
+from typing import BinaryIO, List
+
+import numpy
+from mpi4py import MPI
+
+from lute.execution.ipc import Message
+from lute.io.models.base import *
+from lute.tasks.task import *
+
+
+class MergeStreamFiles(Task):
```

Comment:
Do you think it may make sense to change the name here to `ConcatenateStreamFiles` or something similar? I'm wondering if this name may lead to some confusion with `merge` `Task`s (reflection merging).  I think there is some ambiguity until you read through the code - same with the docstring.

Reply:
Yes make perfect sense